README.md

```diff
@@ -83,10 +83,12 @@ features = ["extension-module"]
 extern crate pyo3;
 use pyo3::prelude::*;
 
+use pyo3::py::modinit as pymodinit;
+
 // add bindings to the generated python module
 // N.B: names: "librust2py" must be the name of the `.so` or `.pyd` file
 /// This module is implemented in Rust.
-#[py::modinit(rust2py)]
+#[pymodinit(rust2py)]
```

Comment:
What is the reason for this change?

Reply:
`py::modinit` doesn't work without the `proc_macro_path_invoc` feature, which isn't expected to be stabilized soon (unlike proc macros and specialization).

Reply:
I see